File: Examples.VB/Features/Formatting/CellType/AddButtonCellType.vb
Namespace Features.Formatting.CellType
    Public Class AddButtonCellType
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)
            Dim cellType As New ButtonCellType With {
                .Text = "Hello",
                .ButtonBackColor = "Azure",
                .MarginLeft = 10,
                .MarginRight = 10
            }

            worksheet.Range!C5.CellType = cellType
        End Sub
    End Class
End Namespace

File: Examples/Features/Formatting/CellType/AddButtonCellType.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formatting.CellType
{
    public class AddButtonCellType : ExampleBase
    {
        public override bool IsNew => true;
        public override void Execute(Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];
            ButtonCellType cellType = new ButtonCellType
            {
                Text = "Hello",
                ButtonBackColor = "Azure",
                MarginLeft = 10,
                MarginRight = 10
            };

            worksheet.Range["C5"].CellType = cellType;
        }
    }
}
